code/game/objects/items/weapons/RCD.dm

```diff
@@ -218,6 +237,9 @@ RCD
 
 /obj/item/weapon/rcd/New()
 	..()
+
+	no_ammo_message = "<span class='warning'>The \'Low Ammo\' light on \
```

Comment:
Don't do this. Just set the no_ammo_message for each rcd and account for its standard name.
Also god damn please stop with the hideously short line lengths it's actually absurd